help/custom-profile-fields.md

```diff
@@ -56,6 +56,22 @@ There are several different types of fields available.
 * **Person picker**: For selecting one or more users, like "Manager" or
     "Direct reports".
 
+## Users can edit this field
+Some Organizations would find it useful to disallow users changing the values of their custom profile fields.
+
```

Comment:
Change to:

## Configure whether users can edit custom profile fields

{!admin-only.md!}

You can configure whether users in your organization can modify their own custom profile fields. For example, you may want to restrict editing if syncing profile fields from an employee directory.